projects/packages/jetpack-mu-wpcom/webpack.config.js

```diff
@@ -19,7 +22,28 @@ module.exports = {
 		...jetpackConfig.resolve,
 	},
 	node: false,
-	plugins: [ ...jetpackConfig.StandardPlugins() ],
+	plugins: [
+		...jetpackConfig.StandardPlugins( {
+			DependencyExtractionPlugin: { injectPolyfill: true },
+			MiniCssExtractPlugin: { filename: '[name]/[name].css' },
```

Comment:
Outputs CSS in the same build directory as the entry.

For Verbum, css will output in `build/verbum-comments`.